Dockerfile

```diff
@@ -8,7 +8,8 @@ ENV FILTER=""
 ENV IFACE="any"
 ENV MAXFILESIZE="1000"
 ENV MAXFILENUM="10"
-ENV FILENAME="dump"
+ENV FILENAME="dump.pcap"
+ENV FORMAT="pcap"
```

Comment:
As default capture format is `pcapng` at present, I'd rather make that the default, which can be overwritten to enforce `pcap` format, if needed.

Reply:
Done.